crates/aranya-daemon/src/policy.md

```diff
@@ -1663,8 +1962,8 @@ command AssignLabel {
         let author = get_valid_device(envelope::author_id(envelope))
         let target = get_valid_device(this.device_id)
 
-        // Only Owners and Operators can assign labels to Members.
-        check is_owner(author.role) || is_operator(author.role)
+        check device_can_execute_op(author.device_id, "AssignLabel")
+        check author_dominates_target(author.device_id, target.device_id)
```

Comment:
I don't think we care about dominance here. Why does it matter if, for example, an operator grants an admin permission to use a label?